CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+
+<a name="v0.19.0"></a>
+## [v0.19.0](https://github.com/sourcenetwork/defradb/compare/v0.18.0...v0.19.0)
+
+> 2025-08-08
+
+DefraDB v0.19 is a major pre-production release. Until the stable version 1.0 is reached, the SemVer minor patch number will denote notable releases, which will give the project freedom to experiment and explore potentially breaking changes.
+
+To get a full outline of the changes, we invite you to review the official changelog below. This release does include a Breaking Change to existing v0.18.x databases. If you need help migrating an existing deployment, reach out at [hello@source.network](mailto:hello@source.network) or join our Discord at https://discord.gg/w7jYQVJ/.
```

Comment:
question: is this discord link working for you? It seems broken to me.

Reply:
Ironically, the invite link works for me. But, it's the wrong discord.

Fred, you can use the new link: source.network/discord

Reply:
Will do

Reply:
Done. I replaced it in every file it was mentioned.